source/js/components/join/language-select.jsx

```diff
@@ -17,7 +17,7 @@ export default class LanguageSelect extends Component {
       es: `Español`,
       fr: `Français`,
       pl: `Polski`,
-      pt: `Português`,
+      "pt-BR": `Português`,
```

Comment:
@TheoChevalier should this be something like
```suggestion
      "pt-BR": `Português (Brasil)`,
```
???

Reply:
I think we should keep "Português" here, as the newsletter remains "PT", and accepts people speaking all flavors of Portuguese, even if we get an overwhelming majority of subscribers speaking pt-BR.

Reply:
Sounds good to me!